beancount_import/source/schwab_csv.py

```diff
@@ -346,6 +378,31 @@ def get_processed_entry(
                 fees=self.fees,
                 **shared_attrs,
             )
+        if self.action is BrokerageAction.EXPIRED:
+            assert self.quantity is not None
+            amount = Amount(Decimal(0), currency=CASH_CURRENCY) if self.amount is None else self.amount
+            price = Decimal(0) if self.price is None else self.price
+            lot_info = lots.get_sale_lots(account_meta["schwab_account"], self.symbol, self.date, self.quantity)
```

Comment:
This `lot_info` variable is not used.

Reply:
Passed it to Sell().

Reply:
Anything to do with lots I'm leaving for future PRs.